.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: cargo
```

Comment:
can safely bump dependencies using dependabot since any change to the MSRV introduced by dependency updates will be caught in CI